app/views/students/show.html.erb

```diff
@@ -1,5 +1,5 @@
 <h1><%= @student.name %></h1>
 <h2><%= @student.house %></h2>
 <div class="student">
-  <%= image_tag @student.img_url />
+  <%= image_tag("student.img_url") %>
```

Comment:
`student.img_url` needs to be evaluated as Ruby code, not a string. Removing the quotation marks should take care of that.